api/src/app.js

```diff
@@ -8,18 +8,17 @@ const cookieSession = require('cookie-session');
 const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
 const passport = require('passport');
-const csrf = require('csurf');
 const apiRoutes = require('./api');
 const { errorHandler } = require('./middleware');
 const environment = process.env.NODE_ENV || 'dev';
 const app = express();
 
 const limiter = new RateLimit({
-  windowMs: 1 * 60 * 1000,
-  max: 15,
+  windowMs: 5 * 60 * 1000, // Set window to 5 minutes
```

Comment:
Maybe we can have different limiters based on routes? I think its safe to have max 10 requests for any auth route, and the other routes can use this limiter.

Reply:
Good idea!

Reply:
I'm gonna implement this in a future PR because I can't work on the project for a day and need to merge this since notes are broken, but will fix